conformance/reports/STRUCTURE.md

```diff
@@ -0,0 +1,94 @@
+# Conformance Reports structure
+
+## How this folder is structured
+
+This folder stores all the conformance reports, for all the Gateway API Versions
+and is structured as follows:
+
+```text
+|-- conformance/reports
+|   |-- v1.0
+|   |   |-- acme-operator
+|   |   |   |-- README.md
+|   |   |   |-- standard-v2.13-default-report.yaml
+|   |-- v1.1
+|   |   |-- acme-operator
+|   |   |   |-- README.md
+|   |   |   |-- standard-v2.14-default-report.yaml
+|   |   |   |-- standard-v2.14-with-the-lot-report.yaml
+|   |   |   |-- experimental-v2.14-with-the-lot-report.yaml
+|   |   |-- umbrella-operator
+|   |   |   |-- README.md
+|   |   |   |-- standard-v1.8-default-report.yaml
+```
+
+## Rules
+
+As represented above, each Gateway API version contains a set of folders, one for
+each conformant implementation. The implementation is the owner of its folder and
```

Comment:
I lean toward not having this kind of hierarchy as I think we'll end up having an additional layer of folders just for (maybe) a couple of organizations having a couple of projects (I struggle to imagine more than 2 Gateway API projects for each organization). I think instead that having this structure would be easier to navigate:

```text
|-- conformance/reports
|   |-- v1.1
|   |   |-- acme-foo
|   |   |   |-- README.md
|   |   |   |-- standard-v2.14-default-report.yaml
|   |   |   |-- standard-v2.14-with-the-lot-report.yaml
|   |   |   |-- experimental-v2.14-with-the-lot-report.yaml
|   |   |-- acme-bar
|   |   |   |-- README.md
|   |   |   |-- standard-v1.8-default-report.yaml
```

Reply:
Alright, and honestly I don't think the decision is irreversible it'll be a _pain_ if we end up wanting to do later but not impossible because it's not user facing in the traditional sense.